routes/controllers/docsController.js

```diff
@@ -1,6 +1,34 @@
+const source = require('rfr');
+const MarkdownIt = require('markdown-it');
+const fs = require('fs');
+
+const { setupCommands } = source('bot/scanner/commandHandler');
+const md = new MarkdownIt();
+
+const gettingStartedMD = fs.readFileSync('docs/GettingStarted.md', 'utf-8');
+const configurationsMD = fs.readFileSync('docs/Configurations.md', 'utf-8');
+const roleReactionsMD = fs.readFileSync('docs/RoleReactions.md', 'utf-8');
+const spotifyPluginMD = fs.readFileSync('docs/SpotifyPlugin.md', 'utf-8');
+const gettingHelpMD = fs.readFileSync('docs/GettingHelp.md', 'utf-8');
+
+function renderMarkdown(command) {
+    return (command.docs) ? md.render(command.docs) : null;
+}
+
 const DocsController = {
     get(req, res) {
-        res.render('docs');
+        const client = {
+        };
+        setupCommands(client);
```

Comment:
What are you trying to do here? `setupCommands()` is only meant to be called once, you can pull those right out of the bot. Since reading from the disk is slow and may become a bottle neck if lots of people start using botomir.

Reply:
docsController() doesnt know about Bot, because our router and website are separate from where the Bot logins. We would have to pass Bot through many different callers and it would be quite ugly. It would also result in more tightly coupled code.

To get around this, I call setupCommands with an empty object to collect all the commands and keep our code loosely coupled. 

Yes, this may become a bottleneck in the future. But we are a long ways off from that. If it gets to that point, we should separate out our website and the Bot instance into two separate repos and deploy them separately.